packages/perseus-editor/src/widgets/python-program-editor.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-useless-escape */
+/**
+ * This editor is for embedding Khan Academy Python programs.
+ */
+
+import {
+    components,
+    Changeable,
+    EditorJsonify,
+} from "@khanacademy/perseus";
+import PropTypes from "prop-types";
```

Comment:
Modern Editors (well, all Perseus components), should not use `prop-types`. Can you convert this to use standard TypeScript `Props` instead, please?